readme.txt

```diff
@@ -22,6 +22,9 @@ This section describes how to install the plugin and get it working.
 
 == Changelog ==
 
+= 2.3.5 =
+* _Enter your changes here_
```

Comment:
Replace this with the changes under `unreleased`

Reply:
Thank you for the catch. I forgot to bring that commit back from the previous one 😄 

Updated it in https://github.com/Automattic/wc-calypso-bridge/pull/1438/commits/971d9a41676e0b7d185599caa7432c19fbfe4631